### ISSUE TITLE
Gnome 41

### ISSUE THREAD

**Issue by apinter** [home-sweet-gnome/dash-to-panel#1475]
Hey,

Been checking out #1468 and it seem work fine and a big thanks to @philippun1 for that. Is it not going to make it's way to the Gnome Extensions?

**Comment by dfkamm** [comment]
Wondering about this too.

**Comment by apinter** [comment]
@hectorfarahani @riekusr you need to use #1468 if you need it if it is still not available through the official gnome extensions page. Been using it ever since and it works like a charm. Little to no elbow grease ;)

**Comment by apinter** [comment]
> Maybe a silly question, but how would I manually install #1468? Would I just git clone into my home directory then make install?

There are no "silly questions". Silly is if you have a question and you don't ask it ;) 
And yes pretty much as you explained. Clone the project and run `make install` and you should be good to go ^_^
The [wiki is also covering this](https://github.com/home-sweet-gnome/dash-to-panel/wiki/Installation#installation-from-github). Hit us up if you need help.

**Comment by ping-wu** [comment]
Clone the project and run `make install` and you should be good to go ^_^ The [wiki is also covering this](https://github.com/home-sweet-gnome/dash-to-panel/wiki/Installation#installation-from-github). Hit us up if you need help.

Install failed.  Got the following message:

ryzen@L32:~/Downloads/dash-to-panel$ make install
glib-compile-schemas ./schemas/
msgfmt -c po/cs.po -o po/cs.mo
make: msgfmt: No such file or directory
make: *** [Makefile:58: po/cs.mo] Error 127

I am using Debian Sid.  Thanks!

**Comment by danieltharp** [comment]
> Clone the project and run `make install` and you should be good to go ^_^ The [wiki is also covering this](https://github.com/home-sweet-gnome/dash-to-panel/wiki/Installation#installation-from-github). Hit us up if you need help.
> 
> Install failed. Got the following message:
> 
> ryzen@L32:~/Downloads/dash-to-panel$ make install glib-compile-schemas ./schemas/ msgfmt -c po/cs.po -o po/cs.mo make: msgfmt: No such file or directory make: *** [Makefile:58: po/cs.mo] Error 127
> 
> I am using Debian Sid. Thanks!

You need to `apt install gettext` as well.

**Comment by ping-wu** [comment]
You need to `apt install gettext` as well.

Thanks. It works great!

**Comment by philippun1** [comment]
@mbevc1 Yes all the older releases will still work the same way as before. I checked and it seems F34 will be supported 4 weeks after the release of F35, so gnome 40 is obsolete soon as well for Fedora.
IMHO it would probably be best to release one last version with all the latest gnome 40 fixes (i.e. show applications bug) and then go forward with gnome 41+ only.
@charlesg99 What is your opinion on that?

**Comment by charlesg99** [comment]
@philippun1 I'm not sure about this, I always thought trying to support multiple gnome versions would be beneficial in the long run and reduce confusion, but I've never tried the alternative of branching per version. I see other extensions that only support 1 gnome version at a time and they seem to be doing just fine. At this point the extension code is pretty bloated with the 3.18 to 3.38 support (especially the es5 -> es6 migration) and could really use a good clean up if we move to a 1 version support.

**Comment by mbevc1** [comment]
F35 was released with Gnome41 yesterday :partying_face: . Is there anything blocking us to release Gnome41 compatible version?

**Comment by sharma** [comment]
> EDIT: Never mind, fixed it. After changing the version number, I had to delete the pending "update" before logging out, now it's working perfectly.

@dfkamm How did you delete the pending update? Experiencing the same issue myself and not seeing an option to cancel the update in the Extensions app or extensions.gnome.org.

**Comment by dfkamm** [comment]
> > EDIT: Never mind, fixed it. After changing the version number, I had to delete the pending "update" before logging out, now it's working perfectly.
> 
> @dfkamm How did you delete the pending update? Experiencing the same issue myself and not seeing an option to cancel the update in the Extensions app or extensions.gnome.org.

You need to go to ~/.local/share/gnome-shell/extension-updates and delete the dash-to-panel folder there.

**Comment by sharma** [comment]
That did the trick, thanks!

**Comment by thmo** [comment]
53660a07b removed gnome-shell 41 support from master. why?

**Comment by jubalfh** [comment]
@philippun1 point of order; impish will be supported until july next year; that is not ‘soon’ by any reasonable stretch, and only three months shorter than the estimated eol of fedora 35.

**Comment by philippun1** [comment]
@jubalfh It's still short, 22.04 will be release in April already too. If i.e. Debian 11 would not have sticked to Gnome 3.38 and gone for 40 directly, now that would have been a long support period.
@thmo Because the AppIconMenu -> AppMenu transition in Gnome 41 broke compatibility with Gnome 40.

DTP will not stop working on Distributions with Gnome 40, @charlesg99 just tagged a new release yesterday with the latest fixes for it.

**Comment by mbevc1** [comment]
> @mbevc1 Yes all the older releases will still work the same way as before. I checked and it seems F34 will be supported 4 weeks after the release of F35, so gnome 40 is obsolete soon as well for Fedora.
> IMHO it would probably be best to release one last version with all the latest gnome 40 fixes (i.e. show applications bug) and then go forward with gnome 41+ only.

Was that release yesterday the last one and we can expect next one to support 41? Thanks!
